---
layout: page
title: Grigoriy A. Armeev
permalink: /garmeev
---

##  <b>Grigoriy A. Armeev</b> - Curriculum Vitae

<table class="rowu">
<colgroup>
<col width="20%" />
<col width="80%" />
</colgroup>
<tbody>
	<tr class="rowu">
		<td markdown="span">
			<img src="{{ "/assets/photo_gaa.jpg" | relative_url }}" width="200px" />
		</td>
		<td>
			<table>
				<colgroup>
				<col width="50%" />
				<col width="50%" />
				</colgroup>
				<tr>
					<td>
						<table>
							<tr><td align="right">Website:</td><td><a href="http://intbio.org">http://www.intbio.org</a></td></tr>
							<tr><td align="right">GitHub:</td><td><a href="https://github.com/molsim">molsim</a></td></tr>

							<tr>
								<td align="right">Social media:</td>
								<td valign="top" align="left">
									<div> <a href="https://twitter.com/sa1ary?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-count="false">Follow @2AlexSha</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> </div>

									<a target="_blank" title="find us on Facebook" href="https://www.facebook.com/profile.php?id=100001424042362"> <img src="http://login.create.net/images/icons/user/facebook_40x40.png" width="36" height="20" style="border:1px solid white" alt="follow me on facebook"/> </a>
			
									<a target="_blank" title="find us on VK" href="https://vk.com/g.armeev"> <img src="{{ "/assets/vk.png" | relative_url }}" width="36" height="20" style="border:1px solid white" alt="follow me on VK"/> </a>
			
								</td>
							</tr>
							<tr><td align="right">GoogleScholar:</td><td><a href="https://scholar.google.com/citations?user=eL0KU9wAAAAJ">Grigoriy Armeev</a></td></tr>
							<tr><td align="right">Pubmed:</td><td><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=armeev+g%5Bauth%5D">Armeev G</a></td></tr>
							<tr><td align="right">Scopus ID:</td><td><a href="https://www.scopus.com/authid/detail.uri?authorId=56872901900">56872901900</a></td></tr>
							<tr><td align="right">Istina:</td><td><a href="https://istina.msu.ru/profile/satary/">9571655</a></td></tr>
						</table>
					</td>
				</tr>
			</table>
		</td>
	</tr>
	<tr class="rowu">
		<td valign="top"><b>CONTACT</b></td>
		<td>
			<table>
				<tr><td align="right">E-mail:</td><td><a href="mailto:armeev@intbio.org">armeev@intbio.org</a></td></tr>
				<tr><td align="right">Tel:</td><td>+7 495 939 23 74</td></tr>
				<tr><td align="right">Skype:</td><td>Армеев Григорий</td></tr>
				<tr><td align="right">Address:</td><td>Bioengineering Department, Faculty of Biology, Moscow State University, 1 Leninskie Gory, building 73, Moscow, 119991 Russia</td></tr>
			</table>
		</td>
	</tr>
	<tr class="rowu">
		<td valign="top"><b>CAREER<br>BIOGRAPHY</b></td>
		<td>
to fill this
    </td>
	</tr>
	<tr class="rowu">
		<td valign="top"><b>RESEARCH<br>INTERESTS</b></td>
		<td>
			Our broad aim is on integrating and applying modern concepts and approaches from physical and engineering sciences, molecular modeling, computer science and bioinformatics towards solving problems in biology and bioengineering. Particularly, our current focus is on studying chromatin structure at nucleosomal level via a combination of biophysical, modeling and bioinformatics approaches. We are interested in how nucleosome structure, positioning and dynamics affect interactions with other chromatin proteins and ultimately affect gene expression and control cell functioning at the epigenetic level.

		</td>
	</tr>

	<tr class="rowu">
		<td valign="top"><b>WORK<br>EXPERIENCE</b></td>
		<td>
			<table class="rowu">	
				<tr class="rowu"><td align="center">2014-</td><td align="center"><b>Research fellow</b></td><td>Moscow State University, Faculty of Biology, Department of Bioengineering; focus on integrative moddeling approaches of chromatin structures</td></tr>
				<tr class="rowu"><td align="center">2012-</td><td align="center"><b>System administrator</b></td><td>Moscow State University, Faculty of Biology, Department of Bioengineering, Creating and administratig HPC clusters</td></tr>
			</table>
		</td>
	</tr>
	<tr class="rowu">
		<td valign="top"><b>EDUCATION/<br>DEGREES</b></td>
		<td>
			<table class="rowu">	
				<tr class="rowu"><td align="center">2016</td><td align="center"><b>Certificate</b></td><td>Advanced Studies in Technology Transfer, FAES Graduate School at National Institues of Health, USA (15 credits, GPA 4.0, max 4.0)</td></tr>
				<tr class="rowu"><td align="center">2014--2016</td><td align="center"><b></b></td><td><a href="https://faes.org/content/advanced-studies-in-technology-transfer">Advanced Studies in Technology Transfer</a>, FAES Graduate School at National Institues of Health, USA</td></tr>
				<tr class="rowu"><td align="center">2011</td><td align="center"><b>Dr. Rer. Nat. degree (PHD)</b></td><td>University of Ulm, Germany; Thesis title “Computer simulations of self-assembling nanofibers from thiophene-peptide oligomers”; Magna cum laude; Advisor Prof. A.R. Khokhlov</td></tr>
				<tr class="rowu"><td align="center">2010</td><td align="center"><b>PHD degree (physics and mathematics)</b></td><td>Moscow State University, Russia; Thesis title “Computer simulations and statistical analysis of self-organizing molecular structures based on peptides”; Advisor Prof. A.R. Khokhlov</td></tr>
				<tr class="rowu"><td align="center">2007--2010</td><td align="center"><b>PHD student</b></td><td>Moscow State University, Faculty of Physics, Chair of Physics of Polymers and Crystals</td></tr>
				<tr class="rowu"><td align="center">2007--2008</td><td align="center"><b></b></td><td>“Innovation Studio” Entrepreneurship Study Program at Moscow State University</td></tr>
				<tr class="rowu"><td align="center">2007</td><td align="center"><b>Master of Science degree</b></td><td>Moscow State M.V. Lomonosov University, Faculty of Physics, Chair of Physics of Polymers and Crystals, diploma with excellence (GPA 5.0, max 5.0), Master Thesis “Study of hydration and adsorption of amino acid side chains via MD simulations”; Advisors Prof. A.R. Khokhlov, Prof. V.A. Ivanov. Selected as the best thesis at the department.</td></tr>
				<tr class="rowu"><td align="center">2001--2007</td><td align="center"><b>Student</b></td><td>Moscow State M.V. Lomonosov University, Faculty of Physics</td></tr>
				<tr class="rowu"><td align="center">2002--2005</td><td align="center"><b></b></td><td>Russian-German Institute of Science and Culture at MSU</td></tr>
				<tr><td align="center">1991--2001</td><td align="center"><b>School student</b></td><td>Moscow Comprehensive School No. 1260 with advanced studies of English language. School-leaving certificate with golden medal (GPA 5.0, max 5.0)</td></tr>
			</table>
		</td>
	</tr>

	<tr class="rowu">
		<td valign="top"><b>AWARDS<br>SCHOLARSHIPS</b></td>
		<td>
			<table class="rowu">	
				<tr class="rowu"><td align="center">2013</td><td align="center"><b></b></td><td>Scholarship of US-Russia Collaboration in the Biomedical Sciences NIH Visiting Fellows Program</td></tr>
				<tr class="rowu"><td align="center">2013</td><td align="center"><b></b></td><td>Finalist of the 37-th competition of scientific works by young scientists of Lomonosov Moscow state University</td></tr>
				<tr class="rowu"><td align="center">2012</td><td align="center"><b></b></td><td>Finalist of the contest of works by talented students, graduates and young scientists of Lomonosov Moscow State University established by O.V. Deripaska</td></tr>
				<tr class="rowu"><td align="center">2008</td><td align="center"><b></b></td><td>BioCamp 2008, Novartis International Biotechnology Leadership Camp, Hong Kong: leader of the best Best Performing Team, 2nd Runner Up Best Performing Student</td></tr>
				<tr class="rowu"><td align="center">2008</td><td align="center"><b></b></td><td>Russian Innovation Contest, participant of the 2nd round</td></tr>
				<tr class="rowu"><td align="center">2005</td><td align="center"><b></b></td><td>Scholarship of Vladimir Potanin Foundation</td></tr>
				<tr class="rowu"><td align="center">2005</td><td align="center"><b></b></td><td>Scholarship “Grant of Moscow”</td></tr>
			</table>
		</td>
	</tr>
<!-- <tr class="rowu">
		<td valign="top"><b>PROFESSIONAL<br>ACTIVITIES</b></td>
		<td>
			<table>
				<tr><td align="right">E-mail:</td><td>shaytan_akATmail.bio.msu.ru</td></tr>
				<tr><td align="right">Tel:</td><td>+7 495 939 23 74</td></tr>
				<tr><td align="right">Fax:</td><td>+7 495 939 57 38</td></tr>
				<tr><td align="right">Skype:</td><td>alexeyshaytan</td></tr>
				<tr><td align="right">Address:</td><td>Bioengineering Department, Faculty of Biology, Moscow State University, 1 Leninskie Gory, building 73, Moscow, 119991 Russia</td></tr>
			</table>
		</td>
	</tr>
	<tr class="rowu">
		<td valign="top"><b>SKILLS</b></td>
		<td>
			<table>
				<tr><td align="right">E-mail:</td><td>shaytan_akATmail.bio.msu.ru</td></tr>
				<tr><td align="right">Tel:</td><td>+7 495 939 23 74</td></tr>
				<tr><td align="right">Fax:</td><td>+7 495 939 57 38</td></tr>
				<tr><td align="right">Skype:</td><td>alexeyshaytan</td></tr>
				<tr><td align="right">Address:</td><td>Bioengineering Department, Faculty of Biology, Moscow State University, 1 Leninskie Gory, building 73, Moscow, 119991 Russia</td></tr>
			</table>
		</td>
	</tr>
	<tr class="rowu">
		<td valign="top"><b>STUDENT<br>SUPERVISION</b></td>
		<td>
			<table>
				<tr><td align="right">E-mail:</td><td>shaytan_akATmail.bio.msu.ru</td></tr>
				<tr><td align="right">Tel:</td><td>+7 495 939 23 74</td></tr>
				<tr><td align="right">Fax:</td><td>+7 495 939 57 38</td></tr>
				<tr><td align="right">Skype:</td><td>alexeyshaytan</td></tr>
				<tr><td align="right">Address:</td><td>Bioengineering Department, Faculty of Biology, Moscow State University, 1 Leninskie Gory, building 73, Moscow, 119991 Russia</td></tr>
			</table>
		</td>
	</tr>
	<tr class="rowu">
		<td valign="top"><b>GRANTS</b></td>
		<td>
			<table>
				<tr><td align="right">E-mail:</td><td>shaytan_akATmail.bio.msu.ru</td></tr>
				<tr><td align="right">Tel:</td><td>+7 495 939 23 74</td></tr>
				<tr><td align="right">Fax:</td><td>+7 495 939 57 38</td></tr>
				<tr><td align="right">Skype:</td><td>alexeyshaytan</td></tr>
				<tr><td align="right">Address:</td><td>Bioengineering Department, Faculty of Biology, Moscow State University, 1 Leninskie Gory, building 73, Moscow, 119991 Russia</td></tr>
			</table>
		</td>
	</tr> -->
	<tr class="rowu">
		<td valign="top"><b>PUBLICATIONS</b></td>
		<td>
			<table class="rowu">
			<colgroup>
			<col width="5%" />
			<col width="85%" />
			</colgroup>
				<tr><td align="center" colspan="2"><a href="https://goo.gl/6W1QKS" style="background-color: #b7f3b7;">GoogleScholar</a> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=shaytan+a%5Bauth%5D" style="background-color: #b7f3ee;"> Pubmed</a> <a href="https://paperpile.com/shared/ti7t8P" style="background-color: #ff97ec">Complete list of publications with PDFs</a></td></tr>
			<tr><td align="center" colspan="2"><b>Selected papers</b></td></tr>
			<!-- https://docs.google.com/document/d/e/2PACX-1vRTwm9QaWFJeoQ1PuP7I9X4rAyMig-QG2KzqFKglMvmQlAXuw0mbjdQYqpUFsFhutiQw_8C549aQA-g/pub -->
			<tr class="rowu"><td>2017</td><td align="left">Xiao H, Wang F*, Wisniewski J*, <u>Shaytan AK*</u>, Ghirlando R, FitzGerald PC, et al. Molecular basis of CENP-C association with the CENP-A nucleosome at yeast centromeres. <b>Genes Dev.</b> 2017; doi:<a href="http://dx.doi.org/10.1101/gad.304782.117">10.1101/gad.304782.117</a>; *-equal contribution</td></tr>

			<tr class="rowu"><td>2017</td><td align="left"><u>Shaytan AK*</u>, Xiao H*, Armeev GA, Wu C, Landsman D, Panchenko AR. Hydroxyl-radical footprinting combined with molecular modeling identifies unique features of DNA conformation and nucleosome positioning. <b>Nucleic Acids Res.</b> 2017;45: 9229–9243. doi:<a href="http://dx.doi.org/10.1093/nar/gkx616">10.1093/nar/gkx616</a>; *-equal contribution</td></tr>

			<tr class="rowu"><td>2017</td><td align="left">El Kennani S, Adrait A, <u>Shaytan AK</u>, Khochbin S, Bruley C, Panchenko AR, et al. MS_HistoneDB, a manually curated resource for proteomic analysis of human and mouse histones. <b>Epigenetics Chromatin</b>. 2017;10: 2. doi:<a href="http://dx.doi.org/10.1186/s13072-016-0109-x">10.1186/s13072-016-0109-x</a></td></tr>

			<tr class="rowu"><td>2016</td><td align="left">Draizen EJ*, <u>Shaytan AK*</u>, Marino-Ramirez L, Talbert PB, Landsman D, Panchenko AR. HistoneDB 2.0: a histone database with variants--an integrated resource to explore histones and their variants. <b>Database-the Journal of Biological Databases and Curation</b>. 2016;2016. doi:<a href="http://dx.doi.org/10.1093/database/baw014">10.1093/database/baw014</a>; *-equal contribution</td></tr>

			<tr class="rowu"><td>2016</td><td align="left"><u>Shaytan AK</u>, Armeev GA, Goncearenco A, Zhurkin VB, Landsman D, Panchenko AR. Coupling between Histone Conformations and DNA Geometry in Nucleosomes on a Microsecond Timescale: Atomistic Insights into Nucleosome Functions. <b>J Mol Biol</b>. 2016;428: 221–237. doi:<a href="http://dx.doi.org/10.1016/j.jmb.2015.12.004">10.1016/j.jmb.2015.12.004</a></td></tr>

			<tr class="rowu"><td>2016</td><td align="left">Valieva ME, Armeev GA, Kudryashova KS, Gerasimova NS, <u>Shaytan AK</u>, Kulaeva OI, et al. Large-scale ATP-independent nucleosome unfolding by a histone chaperone. <b>Nat Struct Mol Biol</b>. 2016;23: 1111–1116. doi:<a href="http://dx.doi.org/10.1038/nsmb.3321">10.1038/nsmb.3321</a></td></tr>

			<tr class="rowu"><td>2015</td><td align="left"><u>Shaytan AK</u>, Landsman D, Panchenko AR. Nucleosome adaptability conferred by sequence and structural variations in histone H2A–H2B dimers. <b>Curr Opin Struct Biol</b>. Elsevier; 2015;32C: 48–57. doi:<a href="http://dx.doi.org/10.1016/j.sbi.2015.02.004">10.1016/j.sbi.2015.02.004</a></td></tr>

			<tr class="rowu"><td>2014</td><td align="left">Nishi H, <u>Shaytan A</u>, Panchenko AR. Physicochemical mechanisms of protein regulation by phosphorylation. <b>Front Genet</b>. 2014;5: 270. doi:<a href="http://dx.doi.org/10.3389/fgene.2014.00270">10.3389/fgene.2014.00270</a></td></tr>

			<tr class="rowu"><td>2013</td><td align="left">Yolamanova M, Meier C, <u>Shaytan AK</u>, Vas V, Bertoncini CW, Arnold F, et al. Peptide nanofibrils boost retroviral gene transfer and provide a rapid means for concentrating viruses. <b>Nat Nanotechnol</b>. 2013;8: 130–136. doi:<a href="http://dx.doi.org/10.1038/nnano.2012.248">10.1038/nnano.2012.248</a></td></tr>

			<tr class="rowu"><td>2011</td><td align="left"><u>Shaytan AK</u>, Schillinger E-K, Khalatur PG, Mena-Osteritz E, Hentschel J, Börner HG, et al. Self-assembling nanofibers from thiophene-peptide diblock oligomers: a combined experimental and computer simulations study. <b>ACS Nano</b>. ACS Publications; 2011;5: 6894–6909. doi:<a href="http://dx.doi.org/10.1021/nn2011943">10.1021/nn2011943</a></td></tr>

			<tr class="rowu"><td>2010</td><td align="left"><u>Shaytan AK</u>, Ivanov VA, Shaitan KV, Khokhlov AR. Free energy profiles of amino acid side chain analogs near water-vapor interface obtained via MD simulations. <b>J Comput Chem.</b> Wiley Online Library; 2010;31: 204–216. doi:<a href="http://dx.doi.org/10.1002/jcc.21267">10.1002/jcc.21267</a></td></tr>

			<tr class="rowu"><td>2009</td><td align="left"><u>Shaytan AK</u>, Shaitan KV, Khokhlov AR. Solvent accessible surface area of amino acid residues in globular proteins: correlation of apparent transfer free energies with experimental hydrophobicity scales. <b>Biomacromolecules</b>. 2009;10: 1224–1237. doi:<a href="http://dx.doi.org/10.1021/bm8015169">10.1021/bm8015169</a></td></tr>

			<tr><td align="center" colspan="2"><b>Theses/Dissertations</b></td></tr>
			
			<tr class="rowu"><td>2011</td><td align="left"><u>Shaytan AK</u>. Computer simulations of self-assembling nanofibers from thiophene-peptide oligomers. PHD thesis. Universität Ulm. 2012. doi:<a href="http://dx.doi.org/10.18725/OPARU-1936">10.18725/OPARU-1936</a></td></tr>

			<tr class="rowu"><td>2010</td><td align="left"><u>Shaytan AK</u>. Computer simulations and statistical analysis of self-organizing molecular structures based on peptides (in Russian). PHD thesis. Moscow State University. 2010. <a href="https://www.dropbox.com/s/pbpavd74fnuciox/thesis_final_sm.pdf?dl=0">PDF</a></td></tr>

			<tr class="rowu"><td>2007</td><td align="left"><u>Shaytan AK</u>. Study of hydration and adsorption of amino acid side chains via MD simulations (in Russian). Master thesis. Moscow State University. 2007. <a href="https://www.dropbox.com/s/tyomx1l539jx2ll/diplom_final.pdf?dl=0">PDF</a></td></tr>


			</table>
		</td>
	</tr>
	<tr class="rowu">
		<td valign="top"><b>PATENTS/<br>SOFTWARE<br>REGISTRATION</b></td>
		<td>
			<table>
				<tr><td align="right"></td><td><a href="https://patentscope.wipo.int/search/en/detail.jsf?docId=RU161773295&recNum=3&maxRec=44&office=&prevFilter=&sortOption=&queryString=p53+and+mutant&tab=NationalBiblio">&bull; (RU0002580006)</a> METHOD FOR SCREENING OF POTENTIAL ANTICANCER PREPARATIONS - FACT INHIBITORS</td></tr>
				<tr><td align="right"></td><td><a href="http://www1.fips.ru/fips_servl/fips_servlet?DB=EVM&DocNumber=2013616706&TypeFile=html">&bull; Software suite for reconstruction of spatial structures of proteins and complex based on low-resolution electron density maps (in Russian)</a></td></tr>

			</table>
		</td>
	</tr>
<!-- <tr class="rowu">
		<td valign="top"><b>TALKS</b></td>
			<td>
			<table>
				<tr><td align="right">E-mail:</td><td>shaytan_akATmail.bio.msu.ru</td></tr>
				<tr><td align="right">Tel:</td><td>+7 495 939 23 74</td></tr>
				<tr><td align="right">Fax:</td><td>+7 495 939 57 38</td></tr>
				<tr><td align="right">Skype:</td><td>alexeyshaytan</td></tr>
				<tr><td align="right">Address:</td><td>Bioengineering Department, Faculty of Biology, Moscow State University, 1 Leninskie Gory, building 73, Moscow, 119991 Russia</td></tr>
			</table>
		</td>
	</tr> -->
	<tr class="rowu">
		<td valign="top"><b>REFERENCES</b></td>
		<td>
			<table class="rowu">
			<colgroup>
			<col width="40%" />
			<col width="40%" />
			</colgroup>
				<tr class="rowu">
					<td align="left" valign="top"><a href="https://polly.phys.msu.ru/en/personnel/khokhlov.html"><b>Prof. Alexei R. Khokhlov</b></a> (PHD advisor)<br>Member of Russian Academy of Sciences<br>Chief, Chair of Polymer and Crystal Physics<br>Moscow State University, Russia </td>
					<td align="left" valign="top"><a href="https://www.ncbi.nlm.nih.gov/CBBresearch/Panchenko/contact.html"><b>Dr. Anna R. Panchenko</b></a> (Post-doctoral advisor)<br>Lead Scientist<br>National Center for Biotechnology<br> Information, NIH, USA</td>
				</tr>
				<tr class="rowu">
					<td align="left" valign="top"><a href="https://www.ncbi.nlm.nih.gov/research/staff/landsman/"><b>Dr. David Landsman</b></a><br>Chief, Computational Biology Branch<br>National Center for Biotechnology<br> Information, NIH, USA</td>
					<td align="left" valign="top"><a href="http://bio.jhu.edu/directory/carl-wu/"><b>Prof. Carl Wu</b></a><br>Member of National Academy of Sciences<br> Department of Biology, Johns Hopkins University, USA</td>
				</tr>
				<tr class="rowu">
					<td align="left" valign="top"><a href="https://www.uni-ulm.de/en/nawi/instituteofadvancedenergyrelatednanomaterials/staff/"><b>Prof. Pavel G. Khalatur</b></a><br>Institute of Advanced Energy Related<br> Nanomaterials, University of Ulm, Germany</td>
				</tr>
			</table>
		</td>
	</tr>
</tbody>
</table>
